Header header
int8 task_id
task_geometry task_geometry
int8 allocated
int8 type
